src/lib/account_update.ts

```diff
@@ -1899,10 +1899,13 @@ function addMissingSignatures(
       let i = additionalPublicKeys.findIndex((pk) =>
         pk.equals(accountUpdate.body.publicKey).toBoolean()
       );
-      if (i === -1)
-        throw Error(
-          `addMissingSignatures: Cannot add signature for ${accountUpdate.publicKey.toBase58()}, private key is missing.`
-        );
+      if (i === -1) {
+        // private key is missing, but we are not throwing an error here
+        // there is a change signature will be added by the wallet
+        // if not, error will be thrown by verifyAccountUpdate
+        // while .send() execution
+        return accountUpdate as AccountUpdate & { lazyAuthorization?: LazyProof };
```

Comment:
you're lying about the type here, I'm not sure if that's a good idea :/ Could you instead at least set the signature to an empty string or dummy signature, and use `Authorization.setSignature`? that way, at least the types are correct

Reply:
I pushed an update, thanks for noticing this! I just copy-pasted badly...

Reply:
thanks, looks good to merge now!